static/css/main.css

```diff
@@ -7,68 +7,127 @@ body {
   font-size: 16px;
   font-family: 'Lato', sans-serif; }
 
-.container {
-  width: 100%; }
+.encloser {
+  display: flex;
+  min-height: 100vh;
```

Comment:
@jimmcgaw I'm not how to say this, but is `min-height: 100vh` "flexbox-friendly"?